src/gsy_e/gsy_e_core/rq_job_handler.py

```diff
@@ -77,6 +72,20 @@ def launch_simulation_from_rq_job(scenario: Dict,
             "aggregator_device_mapping": aggregator_device_mapping
         }
 
+        assert isinstance(scenario, dict)
+        gsy_e.constants.CONFIGURATION_ID = scenario.pop("configuration_uuid")
+        if "collaboration_uuid" in scenario or settings.get("type") in [
+                ConfigurationType.CANARY_NETWORK.value, ConfigurationType.B2B.value]:
+            gsy_e.constants.EXTERNAL_CONNECTION_WEB = True
+            GlobalConfig.IS_CANARY_NETWORK = scenario.pop("is_canary_network", False)
+            gsy_e.constants.RUN_IN_REALTIME = GlobalConfig.IS_CANARY_NETWORK
+
+            if settings["type"] == ConfigurationType.B2B.value:
+                ConstSettings.ForwardMarketSettings.ENABLE_FORWARD_MARKETS = True
+                # Disable fully automatic trading mode for the template strategies in favor of
+                # UI manual and auto modes.
+                ConstSettings.ForwardMarketSettings.FULLY_AUTO_TRADING = False
```

Comment:
I understand that in the case of `ConstSettings.ForwardMarketSettings.FULLY_AUTO_TRADING = True`, the template strategies will trade automatically (like they do also in the "normal" simulations. However, what is the UI auto mode? I thought that this is exactly this case? Or is the UI actually sending orders automatically?

Reply:
This is one thing that was not clear from the requirements. The FULL_AUTO_TRADING is essentially only used in the CLI (where the template strategies are posting automatically bids and offers on all open market slots). This does not apply to the UI, since both auto and manual trading are only posting orders on specific market slots (according to the start / end time). The difference between the manual and the auto mode, is that in the auto mode, the order is updating its energy rate during the market slot, contrary to manual mode, where the order never updates its energy rate.

Reply:
I think I understand, let me summarise:
1. cli: FULLY_AUTO_TRADING = True: (default from gsy-framework), orders are posted on all markets automatically (handled [here](https://github.com/gridsingularity/gsy-e/pull/1556/files#diff-808df61501ae0753a81cf36ebb1d3ced3f1cacf3a0f07b870b1f8b0c7c42a784R123))
2. UI auto mode: orders are updated in a specific time frame (implemented [here](https://github.com/gridsingularity/gsy-e/pull/1556/files#diff-b79d23882e80bccfa3f0e1810199e0825122845183b2e1ade1bb7d9d9fea325dR63) )
3. UI manual mode: orders are updated in a specific time frame ones (implemented [here](https://github.com/gridsingularity/gsy-e/pull/1556/files#diff-b79d23882e80bccfa3f0e1810199e0825122845183b2e1ade1bb7d9d9fea325dR108))

Reply:
You got it, but I would rephrase #3 to "orders are only posted once and never updated."